src/indicators/flooding/glofas/utils.py

```diff
@@ -265,13 +259,12 @@ def get_crps(
                     thresh_to_use = thresh
                 idx = observations > thresh_to_use
                 forecast, observations = forecast[:, idx], observations[idx]
-            crps = (
-                xs.crps_ensemble(
-                    observations, forecast, member_dim="number"
-                ).values
-                / norm
+
+            df_crps.loc[leadtime, station] = calc_crps(
+                observations,
+                forecast,
+                normalization=normalization,
```

Comment:
So I guess in this case, the normalization would be calculated after any threshold cut and not on the full sample.

What if you also allowed the `normalization` parameter to be a float, that way you could pass a value directly to use (in addition to the current options), and for the case of GloFAS, just to keep the code working the way it was for now, this value would be calculated in the GloFAS method. Does that make sense?

Reply:
@turnerm you are so creative!

Reply:
Not sure if that's a good thing here, aha